src/mintpy/objects/coord.py

```diff
@@ -68,44 +68,50 @@ def open(self):
             if self.lookup_file:
                 self.lut_metadata = readfile.read_attribute(self.lookup_file[0])
 
-    def lalo2yx(self, coord_in, coord_type):
-        """convert geo coordinates into radar coordinates for Geocoded file only
-        Parameters: geoCoord   - list / tuple / 1D np.ndarray / float, coordinate(s) in latitude or longitude
-                    metadata   - dict, dictionary of file attributes
-                    coord_type - str, coordinate type: latitude or longitude
-        Example:    300 = coordinate.lalo2yx(32.104990,    metadata,'lat')
-                    [1000,1500] = coordinate.lalo2yx([130.5,131.4],metadata,'lon')
-        """
-        self.open()
-        if not self.geocoded:
-            raise ValueError('Input file is not geocoded.')
-
+    def _clean_coord(self, coord_in):
         # input format
         if isinstance(coord_in, np.ndarray):
             coord_in = coord_in.tolist()
         # note: np.float128 is not supported on Windows OS, use np.longdouble as a platform neutral syntax
         if isinstance(coord_in, (float, np.float16, np.float32, np.float64, np.longdouble)):
```

Comment:
If we go with the pair style, let's add the integer type here as well, so that we can reuse this func in `yx2lalo()`.

Reply:
done!